less/index.less

```diff
@@ -52,8 +52,176 @@ table {
 * {
     box-sizing: border-box;
 }
-
+@font:1.6rem;
```

Comment:
While this is fine here, get it the practice now of more descriptive variable names. Something like `@font-p` or `@font-body`, to give more meaning as to what this size font should be used for. It makes much more sense when working on a larger project where you'll have many different font sizes.